conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/RuleValidators.scala

```diff
@@ -54,4 +58,42 @@ object RuleValidators {
       throw new ValidationException(s"$datasetName - $message $errorMessages" )
     }
   }
+
+  /**
+    * Checks if Spark allows casting from one specific type to another.
+    * Throws an instance of ValidationException exception otherwise.
+    *
+    * @param ruleName        A name of a conformance rule to be used as part of an exception error message
+    * @param inputColumnName A name of an input column to be used as part of an exception error message
+    * @param inputType       A type of an input field
+    * @param outputType      A type to cast to
+    * @param spark           (implicit) A Spark Session
+    */
+  @throws[ValidationException]
+  def validateTypeCompatibility(ruleName: String,
+                                inputColumnName: String,
+                                inputType: DataType,
+                                outputType: String)
```

Comment:
```suggestion
                                outputTypeName: String)
```
`outputType` is little misleading compared to `inputType'